provision-team/svcbusclient/svcbusclient.csproj

```diff
@@ -1,12 +1,12 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
.NET Core 2.0 is over 3 years out of support. Please update this app to .NET 6.0, which is the latest LTS release. That can be handled in a separate PR, if you prefer.